spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -973,6 +973,7 @@ def handle_hover_response(self, contents):
         """Handle hover response."""
         try:
             content = contents['params']
+            self.sig_display_object_info.emit(content)

```

Comment:
Problem when I merged and solved conflicts @ccordoba12. Need to remove this line!

Reply:
Great!!